mongo/integration/client_test.go

```diff
@@ -600,6 +603,99 @@ func TestClient(t *testing.T) {
 		assert.Equal(t, 0, closed, "expected no connections to be closed")
 	})
 
+	mt.Run("RTT90 is monitored", func(mt *mtest.T) {
+		if testing.Short() {
+			t.Skip("skipping integration test in short mode")
+		}
```

Comment:
Instead of including this at the beginning of individual `mt.Run` subtests, we should include this in `mtest.New()`. Check out a proposed PR: https://github.com/mongodb/mongo-go-driver/pull/973